src/backend/core/cart/serializers.py

```diff
@@ -102,16 +107,23 @@ class CartSerializer(ModelSerializer):
     #     write_only=True,
     #     required=False,
     # )
-    # shipping_info = ShippingInfoSerializer(read_only=True)
-    # billing_info = BillingInfoSerializer(read_only=True)
+    shipping_info = ShippingInfoSerializer(read_only=True)
```

Comment:
maybe you should use separate seializer (which can inherit from this one), 
`CartSeriailzer` is now used for getting cart data, so this change means that we're kind of overfetching information that we don't need there (billing & shipping info is on a separate page).

Reply:
Sorry, this was left here by an accident 😊 . Check it now